qemu/tests/hotplug_mem.py

```diff
@@ -79,10 +79,12 @@ def start_test(self):
         try:
             if stage != "after":
                 sub_test = sub_test_runner()
-                func(vm, target_mem)
+                for target_mem in target_mems.split():
+                    func(vm, target_mem)
                 self.check_memory(vm)
```

Comment:
Do we need to check_memory after hotplug/unplug every time ?

Reply:
better to have check included inside the `for` loop for each targetmem